pkg/providers/v1/aws_loadbalancer.go

```diff
@@ -1234,7 +1238,8 @@ func (c *Cloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadBala
 
 // syncElbListeners computes a plan to reconcile the desired vs actual state of the listeners on an ELB
 // NOTE: there exists an O(nlgn) implementation for this function. However, as the default limit of
-//       listeners per elb is 100, this implementation is reduced from O(m*n) => O(n).
+//
```

Comment:
Is this doc change intentional?

Reply:
Allows new gofmt to pass

Reply:
ah we probably need a separate PR to gofmt the whole repo then so individual PRs don't have to worry about this